### PR TITLE
Add additional potential domain env vars

### PR DESCRIPTION
**Pull request by Jamstah**
Potential variables for the domain are listed in this openstack doc: http://docs.openstack.org/developer/python-openstackclient/authentication.html

### CODE REVIEW

**Comment by Jamstah** [comment]
@axw would you mind reviewing this one?

File: identity/identity.go [hunk 1]
```diff
@@ -120,9 +120,12 @@ var (
 		"OS_TENANT_NAME",
 		"NOVA_PROJECT_ID",
 	}
-	// CredEnvDomainName is used for Credentials.TenantName.
+	// CredEnvDomainName is used for Credentials.DomainName.
 	CredEnvDomainName = []string{
+		"OS_USER_DOMAIN_NAME",
+		"OS_PROJECT_DOMAIN_NAME",
 		"OS_DOMAIN_NAME",
+		"OS_DEFAULT_DOMAIN",
```
Comment (mjs) [inline identity/identity.go:128]: I know very little about OpenStack. Can you explain why you chose these env vars specifically? There's more listed here: http://docs.openstack.org/developer/python-openstackclient/authentication.html#authenticating-using-identity-server-api-v3

**Review by mjs — COMMENTED** [review]
This is probably fine but I just want some clarification.

**Comment by Jamstah** [comment]
Do you mean the _ID ones? Could be a good idea to add them but I don't know goose so it would need testing to check it still works, or if it would need extending to cope.

I choose them based on the open stack doc and my experience, and guessed at a sensible order. I haven't checked which ones the open stack command line reads, or their preference order, that could be another thing to do.

What do you think?

**Comment by mjs** [comment]
Sounds OK to me. Ship it :)

**Review by mjs — APPROVED** [review]

**Comment by wallyworld** [comment]
Closing this PR as support for the extra env vars was recently added in a different PR.

**Pull request closed without merging**